src/main/java/com/checkmarx/flow/service/FlowService.java

```diff
@@ -73,4 +75,19 @@ private void runScanRequest(ScanRequest scanRequest, List<VulnerabilityScanner>
 
         return enabledScanners;
     }
+
+    public void deleteProject(ScanRequest request) {
+
+        Optional<SastScanner> sastScanner = getEnabledScanners(request)
+                .stream().filter(scanner -> scanner instanceof SastScanner)
+                .map(scanner -> ((SastScanner) scanner))
+                .findFirst();
+
+        if(!sastScanner.isPresent()){
+            log.warn("Delete branch for non-SAST scanner is not supported");
```

Comment:
If enabled scanners are e.g. {sca, sast}, then we won't receive this warning. I guess the warning should appear for each enabled scanner except sast. It also makes sense to define a supportsDeletingProjects field in the VulnerabilityScanner interface.